mono/metadata/threads.h

```diff
@@ -52,6 +53,8 @@ extern MONO_API void mono_threads_set_default_stacksize (uint32_t stacksize);
 extern MONO_API uint32_t mono_threads_get_default_stacksize (void);
 
 MONO_API void mono_threads_request_thread_dump (void);
+// Directory to store thread dumps captured by SIGQUIT
+extern void mono_set_thread_dump_dir(gchar* dir);
```

Comment:
@luhenry does this need to be in this header or is there some internal header where it could be moved to? why `extern`?

Reply:
I cannot find a convenient way to set thread dump dir from mini-runtime.